### ISSUE TITLE
sensu client 1.3.0 refuses to start (no memory error)

### ISSUE THREAD

**Issue by jagadeeshops** [sensu/sensu#1844]
OS: centos 7.3
## Expected Behavior
systemctl start sensu-client
should start the application

## Current Behavior
sensu-service[30653]: /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/ffi-1.9.locate closure from pool (NoMemoryError)
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/ffi-1.9.sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/childprocess-0.5.
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/childprocess-0.5.
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/childprocess-0.5.rocess>'
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/childprocess-0.5.)>'
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/site_ruby/2.4.e'
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/site_ruby/2.4.e'
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-spawn-2.2.sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/site_ruby/2.4.e'
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/site_ruby/2.4.e'
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-1.3.sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/site_ruby/2.4.e'
sensu-service[30653]: from /opt/sensu/embedded/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'

## Possible Solution
changed the user and group  from sensu to root in the below file
/lib/systemd/system/sensu-client.service

## Steps to Reproduce (for bugs)
Installed using yum.
Till now it was installing 1.2.1.  From today it started installing 1.3.0. We have this issue in 1.3.0

## Context
tried to add a new client to the exisiting sensu server (1.2.1) opensource environment.

## Your Environment
sensu server (1.2.1)  on centos 7.3
sensu client 1.2.1 (working fine)
sensu client 1.3.0 (error)
no extensions
basic plugins

**Comment by amdprophet** [comment]
Hi @jagadeeshops,

What CPU architecture are you seeing this issue on? Please run the following:

```
/opt/sensu/embedded/bin/ruby -e "puts RbConfig::CONFIG['host_cpu']"
```

**Comment by portertech** [comment]
The Sensu 1.3.1 packages are not yet in repositories. @jagadeeshops perhaps you are installing 1.3.0-1?

**Comment by portertech** [comment]
@amdprophet this is not related to 1.3.1, the version of sensu-spawn in the error predates our latest change.

**Comment by jagadeeshops** [comment]
@portertech you are right. I am using 1.30

**Comment by jagadeeshops** [comment]
@amdprophet   Architecture is x86_64

**Comment by portertech** [comment]
@jagadeeshops we released Sensu 1.3.3-1 today, it fixes FFI issues that cause OOM and segfaults, please give it a try!

**Comment by jagadeeshops** [comment]
@portertech   Its working now
